.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -3,80 +3,131 @@ on:
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
       - name: Login to GitHub Package Registry
         run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u "${{ github.actor }}" --password-stdin
-      - name: Login to ECR
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.ECR_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.ECR_SECRET }}
-        run: $(aws ecr get-login --no-include-email --region us-east-1)
-      - name: Setup Buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v3
+      - name: Set git parsed values
+        id: vars
+        run: |
+          echo ::set-output name=sha_short::$(git rev-parse --short=8 ${{ github.sha }})
+          echo ::set-output name=tag_name::${GITHUB_REF#refs/tags/}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: operator/docker/base/Dockerfile
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: k8ssandra/cass-operator:${{ steps.vars.outputs.sha_short }}, k8ssandra/cass-operator:latest
+          platforms: linux/amd64
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+  build_system_logger:
+    name: Build System logger Docker Image
+    runs-on: ubuntu-latest
+    needs: testing
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
       - name: Cache Docker layers
         uses: actions/cache@v2
-        id: cache
         with:
           path: /tmp/.buildx-cache
           key: ${{ runner.os }}-buildx-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-buildx-
-      - name: Build and Push Docker
-        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
-        env:
-          ECR_REPO: ${{ secrets.ECR_REPO }}
-          PR_REF: ${{ github.event.pull_request.head.ref }}
-          GITHUB_REPO_OWNER: ${{ github.repository_owner }}
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      - name: Login to GitHub Package Registry
```

Comment:
Are we still pushing images to the GH registry? Changes earlier in the file made me think we are not.

Reply:
We can add it to as the last part of the push/buildx if we want. Don't know if we really need to anymore.

Reply:
If we don't need it, then can we remove this step?